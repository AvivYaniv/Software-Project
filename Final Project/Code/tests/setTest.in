set 1 1 1
edit tests/bgbgbg.txt
print_board
solve tests/bgbgbg.txt
set 0000001 000002 0000000000003
set 00000001 0000001 0000000000006
set 1 1 
set 1 1 1 a
set s 2 2 1
set 2 s 2 2 1
set 2 2 s 1
set 10 1 1
set 1 10 1
set 1 1 10
set 2 1 7
set 2 4 6
set 2 4 7
mark_errors 0
set 4 1 2
solve tests/b2_2_almostComp.txt
autofill
set 1 3 3
undo
solve tests/b2_2_almostComp.txt
autofill
set 1 3 2
set 1 3 4
set 1 3 1
reset
set 1 3 4
autofill 
reset
set 1 3 3
save tests/observed/setTest_save_solve_mode.txt
autofill
autofill
edit tests/b2_2_almostComp.txt
set  	   1          3          5 	 	# this is a very long command this is a very long command this is a very long command this is a very long command this is a very long command this is a very long command this is a very long command this is a very long command this is a very long command this is a very long command
set  	   1          3          4 	 	# this is a very long command this is a very long command this is a very long command this is a very long co
set 1 3 3
save tests/observed/setTest_save_edit_mode.txt
exit
