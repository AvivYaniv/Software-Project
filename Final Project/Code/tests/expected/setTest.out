Sudoku
------
Enter your command:
ERROR: invalid command
Enter your command:
----------------------------------------
|            |          7 |      9   8 |
|  5   4   6 |  8       9 |      3     |
|          8 |          3 |      4     |
----------------------------------------
|            |          5 |      7   9 |
|  6         |          8 |            |
|      8     |          1 |            |
----------------------------------------
|  3   6     |          4 |            |
|  4   5     |  7         |      8   1 |
|  8         |      9   2 |            |
----------------------------------------
Enter your command:
----------------------------------------
|            |          7 |      9   8 |
|  5   4   6 |  8       9 |      3     |
|          8 |          3 |      4     |
----------------------------------------
|            |          5 |      7   9 |
|  6         |          8 |            |
|      8     |          1 |            |
----------------------------------------
|  3   6     |          4 |            |
|  4   5     |  7         |      8   1 |
|  8         |      9   2 |            |
----------------------------------------
Enter your command:
----------------------------------------
|            |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|            |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |          4.|            |
|  4.  5.    |  7.        |      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
Error: cell is fixed
Enter your command:
----------------------------------------
|  6*        |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|            |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |          4.|            |
|  4.  5.    |  7.        |      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
ERROR: invalid command
Enter your command:
----------------------------------------
|  1         |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|            |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |          4.|            |
|  4.  5.    |  7.        |      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
Error: value not in range 0-9
Enter your command:
Error: value not in range 0-9
Enter your command:
Error: value not in range 0-9
Enter your command:
Error: value not in range 0-9
Enter your command:
Error: value not in range 0-9
Enter your command:
Error: value not in range 0-9
Enter your command:
----------------------------------------
|  1   7*    |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|            |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |          4.|            |
|  4.  5.    |  7.        |      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
----------------------------------------
|  1   7*    |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|      6*    |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |          4.|            |
|  4.  5.    |  7.        |      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
----------------------------------------
|  1   7*    |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|      7*    |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |          4.|            |
|  4.  5.    |  7.        |      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
Enter your command:
----------------------------------------
|  1   7     |  2       7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|      7     |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |          4.|            |
|  4.  5.    |  7.        |      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|      3 |  2   1.|
-------------------
|        |  1   2.|
|  2   1 |  3   4 |
-------------------
Enter your command:
Cell <1,2> set to 4
Cell <2,3> set to 4
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|      4 |  1   2.|
|  2   1 |  3   4 |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  3   4 |  1   2.|
|  2   1 |  3   4 |
-------------------
Puzzle solved successfully
Enter your command:
ERROR: invalid command
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|      3 |  2   1.|
-------------------
|        |  1   2.|
|  2   1 |  3   4 |
-------------------
Enter your command:
Cell <1,2> set to 4
Cell <2,3> set to 4
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|      4 |  1   2.|
|  2   1 |  3   4 |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  2   4 |  1   2.|
|  2   1 |  3   4 |
-------------------
Puzzle solution erroneous
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  4   4 |  1   2.|
|  2   1 |  3   4 |
-------------------
Puzzle solution erroneous
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  1   4 |  1   2.|
|  2   1 |  3   4 |
-------------------
Puzzle solution erroneous
Enter your command:
Board reset
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|      3 |  2   1.|
-------------------
|  4     |  1   2.|
|  2   1 |  3   4 |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|      3 |  2   1.|
-------------------
|  4     |  1   2.|
|  2   1 |  3   4 |
-------------------
Enter your command:
Board reset
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|      3 |  2   1.|
-------------------
|  3     |  1   2.|
|  2   1 |  3   4 |
-------------------
Enter your command:
Saved to: tests/observed/setTest_save_solve_mode.txt
Enter your command:
Cell <1,2> set to 4
Cell <2,3> set to 4
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  3   4 |  1   2.|
|  2   1 |  3   4 |
-------------------
Puzzle solved successfully
Enter your command:
ERROR: invalid command
Enter your command:
-------------------
|  1   2 |  4   3 |
|      3 |  2   1 |
-------------------
|        |  1   2 |
|  2   1 |  3   4 |
-------------------
Enter your command:
ERROR: invalid command
Enter your command:
-------------------
|  1   2 |  4   3 |
|      3 |  2   1 |
-------------------
|  4     |  1   2 |
|  2   1 |  3   4 |
-------------------
Enter your command:
-------------------
|  1   2 |  4   3 |
|      3 |  2   1 |
-------------------
|  3     |  1   2 |
|  2   1 |  3   4 |
-------------------
Enter your command:
Saved to: tests/observed/setTest_save_edit_mode.txt
Enter your command:
Exiting...
